specification/containerservice/Fleet.Management/fleetnamespace.tsp

```diff
@@ -120,28 +121,241 @@ union EgressPolicy {
+  DenyAll: "DenyAll",
+
+  @doc("Allow traffic within the same namespace.")
+  AllowSameNamespace: "AllowSameNamespace",
+
+  string,
+}
+
 @added(Versions.v2025_03_02_naar)
-model FleetManagedNamespaceProfile {
-  @doc("Action if Kubernetes managed namespace with the same name already exists.")
-  adoptionPolicy?: AdoptionPolicy;
+@doc("The orchestrator profile to manage the lifecycle of the namespace.")
+model OrchestratorProfile {
+  @doc("The type of the policy to be used.")
+  type: OrchestratorProfileType;
 
-  @doc("Delete options of a fleet managed namespace.")
-  deletePolicy?: DeletePolicy;
+  @doc("The policy configuration for the type of the orchestrator.")
+  policy: OrchestratorPolicy;
+}
 
-  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "Records is the most semantically correct choice for this use-case"
-  @doc("The labels for the fleet managed namespace.")
-  labels?: Record<string>;
+@added(Versions.v2025_03_02_naar)
+@doc("The type of the namespace orchestrator.")
+union OrchestratorProfileType {
+  @doc("Using a fleet hub and kubefleet ClusterResourcePlacement.")
+  Hub: "Hub",
+
+  string,
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("The available orchestrator type.")
+union OrchestratorPolicy {
+  @doc("The Hub Orchestrator.")
+  hubPolicy: HubPolicy,
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("HubPolicy uses a fleet hub and CRP to propagate the namespace.")
+model HubPolicy {
+  @doc("The default ClusterResourcePlacement (CRP) policy configuration.")
+  defaultClusterResourcePlacement: ClusterResourcePlacement;
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("The desired state of ClusterResourcePlacement")
+model ClusterResourcePlacement {
+  @doc("Policy defines how to select member clusters to place the selected resources. If unspecified, all the joined member clusters are selected.")
+  policy?: ClusterResourcePlacementPolicy;
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("ClusterResourcePlacementPolicy contains the rules to select target member clusters to place the selected resources. Only clusters that are both joined and satisfying the rules will be selected. You can only specify at most one of the two fields: ClusterNames and Affinity.")
+model ClusterResourcePlacementPolicy {
+  @doc("Type of placement.")
+  placementType?: PlacementType;
+
+  @doc("ClusterNames contains a list of names of MemberCluster to place the selected resources. Only valid if the placement type is PickFixed.")
+  clusterNames?: string[];
 
+  @doc("Affinity contains cluster affinity scheduling rules. Defines which member clusters to place the selected resources. Only valid if the placement type is PickAll.") // don't forget to update this when we add PickN.
+  affinity?: ClusterResourcePlacementAffinity;
+
+  @doc("If specified, the ClusterResourcePlacement's Tolerations. Tolerations cannot be updated or deleted.")
+  @OpenAPI.extension("x-ms-identifiers", [])
+  tolerations?: CrpToleration[];
```

Comment:
Please make sure we use the same names as the upstream api. It should be Toleration, not CrpToleration. It will be easier to keep the two APIs in sync.

Also avoid using acronyms in field names or data structure names.